python/cudf/cudf/core/multiindex.py

```diff
@@ -998,6 +999,14 @@ def __getitem__(self, index):
         else:
             return result
 
+    @_performance_tracking
+    def equals(self, other) -> bool:
+        return Frame.equals(self, other)
+
+    @_performance_tracking
+    def to_arrow(self) -> pa.Table:
+        return Frame.to_arrow(self)
```

Comment:
We're calling Frame directly because its a multi-column index, right?

Reply:
Yup exactly. For `MutiIndex` there are some methods we need to treat like a `Frame` instead of a `SingleColumnFrame`